tests/lib/rules/jsx-pascal-case.js

```diff
@@ -52,6 +52,8 @@ ruleTester.run('jsx-pascal-case', rule, {
   }, {
     code: '<Modal.Header />'
   }, {
+    code: '<qualification.T3stComp0nent />'
```

Comment:
`T3stComp0nent` isn't in PascalCase; this specific example should fail.

Reply:
If I change it to "T3StComp0Nent", would it be OK? What do you think?

Reply:
yes, i think that would qualify as (a very very weird) PascalCase